recipes/tpot/meta.yaml

```diff
@@ -0,0 +1,60 @@
+{% set name = "tpot" %}
+{% set version = "0.5.2" %}
+{% set md5 = "ad27a30e3d75f25517c0c9bb9159b088" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  fn: {{ name }}-{{ version }}.tar.gz
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz
+  md5: {{ md5 }}
+
+build:
+  number: 0
+  script: python setup.py install --single-version-externally-managed --record record.txt
+  entry_points:
+    - tpot=tpot:main
+
+requirements:
+  build:
+    - python
+    - setuptools
+    - numpy >=1.9.3
+    - pandas >=0.16.2
+    - scikit-learn >=0.16.1
+    - deap >=1.0
+    - tqdm
+    - update_checker
```

Comment:
Probably can drop everything except `python` and `setuptools` from the build dependencies.